homeassistant/components/group/registry.py

```diff
@@ -16,6 +16,23 @@
 
 from .const import DOMAIN, REG_KEY
 
+EXPLICT_SUPPORTED_ON_OFF_DOMAINS = {
```

Comment:
Some of these domains are not documented as supported by the old style groups in the user docs. We should probably add them to the docs list.

https://www.home-assistant.io/integrations/group/#old-style-groups

Reply:
Why are some of the domains in the user docs list not included in this set, eg alarm_control_panel?

Reply:
These domains explicit register their states and their domain will be added to `supported_domains`.

The documentation was updated to the new list already with:

https://github.com/home-assistant/core/pull/116318 and https://github.com/home-assistant/home-assistant.io/pull/32574.

The single state types of the combinations of `on`/`off`, `open`/`closed`, `home`/`not_home` and `locked`/`unlocked`

Reply:
The list only contains the explicit list of domains with the `on`/`off` single state.